doc/architecture/decisions/0028-secure-pipelines-and-control-tower.md

```diff
@@ -0,0 +1,49 @@
+# 28. Secure pipelines and Control tower
+
+[Next >>](9999-end.md)
+
+Date: 2023-06-18
+
+## Status
+
+Proposed
+
+## Context
+
+The existing platform is trying to align with the rest of DI, which uses AWS Control Tower to manage AWS accounts, and
+Secure Pipelines to deploy their systems. As a result, the platform is attempting to align to this flow, while
+also aligning our architecture (as laid out in [ADR 2#](Put link here)). DI also uses an environment per account flow,
```

Comment:
```suggestion
also aligning our architecture (as laid out in [ADR 2#](0002-gdx-share-architecture.md)). DI also uses an environment per account flow,
```

Reply:
this will be the ADR for the DI alignment, not 2